_includes/nav--top.html

```diff
@@ -0,0 +1,53 @@
+
+<header class="usa-header usa-header-basic" role="banner">
```

Comment:
Here is the code for the component. (Borrowed heavily from USWDS-Jekyll)

Reply:
I almost feel like we don't even want to put any links in here? 

We can go the rails config route and just have a commented out example that users can copy. 

That might be going too far from the spec of the project but I think if we truly want something universal we can't even assume the `home` or `external link` values.

On the other hand, there might be some value in providing a something so the user can see what the component looks like locally?

Reply:
I was thinking if we give some defaults then whoever is working with the code can see an example and see how it's working.